warmup.py

```diff
@@ -240,17 +251,39 @@ def accumulate(self, L, points):
         The sample is accumulated into the radial distribution average.
 
         """
-        pass
+        if self.rmax >= L/2:
+            raise ValueError('rmax >= L/2')
+        temp_rdf = np.zeros_like(self._rdf)
+        for i, point in enumerate(points[:-1]):
+            neighbors = np.array(points[i+1:])
+            for neighbor in neighbors:
+                dist_vect = np.absolute(point - neighbor)
+                dist_vect -= L*np.round(dist_vect/L) 
+                dist_mag = np.linalg.norm(dist_vect)
+                if self.rmin <= dist_mag <= self.rmax:
```

Comment:
This is actually a very neat python trick that I did not know about before, so thank you! Other languages do not allow chaining comparison operators together.